src/codec_aom.c

```diff
@@ -876,6 +876,11 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
                 return AVIF_RESULT_UNKNOWN_ERROR;
             }
         }
+
+        if (image->depth == 12) {
+            // The encoder may produce integer overflows with 12-bit input when loop restoration is enabled.
```

Comment:
Please reference the libaom bug crbug.com/aomedia/42302587.

Reply:
Done.